ci-operator/step-registry/ipi/conf/aws/user-min-permissions/ipi-conf-aws-user-min-permissions-commands.sh

```diff
@@ -324,7 +355,7 @@ EOF
 	create_jsoner_py
 	
 	# generate policy file and save it to shared dir so later steps have access to it.
-	cat "${PERMISION_LIST}" | python3 ${JSONER_PY} >"${USER_POLICY_FILE}"
+	cat "${PERMISION_LIST}" | sort | uniq | python3 ${JSONER_PY} >"${USER_POLICY_FILE}"
```

Comment:
For `cat "${PERMISION_LIST}" | sort`, we can simplify it as `sort "${PERMISION_LIST}"`

Reply:
Not a merge blocker. We can live without it, or do it in a followup PR.